File: Selenium/Project/LMS_Activity8.java
package project;


	import java.time.Duration;

import org.openqa.selenium.By;
	import org.openqa.selenium.WebDriver;
	import org.openqa.selenium.WebElement;
	import org.openqa.selenium.chrome.ChromeDriver;
	import org.openqa.selenium.support.ui.WebDriverWait;
	import org.openqa.selenium.support.ui.ExpectedConditions;

	public class LMS_Activity8 {
	    public static void main(String[] args) {
	       
	        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

	        
	        WebDriver driver = new ChromeDriver();

	        try {
	          
	            driver.get("https://alchemy.hguy.co/lms");

	            
	            WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
	            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("nav")));

	            
	            WebElement contactMenuItem = driver.findElement(By.linkText("Contact"));
	            contactMenuItem.click();

	            
	            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("wpforms-8-field_0")));

	          
	            WebElement fullNameField = driver.findElement(By.id("wpforms-8-field_0"));
	            fullNameField.sendKeys("John Doe");

	            WebElement emailField = driver.findElement(By.id("wpforms-8-field_1"));
	            emailField.sendKeys("johndoe@example.com");

	            WebElement messageField = driver.findElement(By.id("wpforms-8-field_3"));
	            messageField.sendKeys("This is a test message for the Contact Us form.");

	          
	            WebElement submitButton = driver.findElement(By.id("wpforms-8-submit"));
	            submitButton.click();

	         
	            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".wpforms-confirmation-container")));

	           
	            WebElement confirmationMessage = driver.findElement(By.cssSelector(".wpforms-confirmation-container"));
	            System.out.println("Confirmation Message: " + confirmationMessage.getText());

	        } catch (Exception e) {
	            e.printStackTrace();
	        } finally {
	            
	            driver.quit();
	        }
	    }
	}

File: Selenium/Project/lms_Activity8.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()

    #Find the menu item that says “contact”” and click it
    driver.find_element(By.XPATH, "//ul[contains(@class,'nav-menu')]/li[4]/a").click()

    # Find the input fields
    full_name = driver.find_element(By.XPATH, "//input[contains(@id,'8-field_0')]") 
    email = driver.find_element(By.XPATH,"//input[contains(@id,'8-field_1')]")
    subject = driver.find_element(By.XPATH,"//input[contains(@id,'8-field_3')]")
    comment = driver.find_element(By.XPATH,"//textarea[contains(@id,'8-field_2')]")

    #pass input details
    full_name.send_keys("Shilpa")
    email.send_keys("test@gmail.com")
    subject.send_keys("Social Media Marketing")
    comment.send_keys("This is a test message")

    #Find submit button and click
    driver.find_element(By.XPATH,"//button[@data-submit-text='Send Message']").click()

    #Find Confirmation message
    success_message = driver.find_element(By.ID,"wpforms-confirmation-8")
    print("The Confirmation message displayed is:", success_message.text)

    # Close the WebDriver
    driver.quit()